.github/ISSUE_TEMPLATE/standard-issue-.md

```diff
@@ -0,0 +1,30 @@
+---
+name: 'Standard Issue '
+about: Standard template for an issue
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## **Description:**
+
+Describe the general demands of the issue.
+
+## **Requirements:**
+
+- [ ] todo 1
+- [ ] todo 2
+- [ ] todo 3
+
+## **External Factors:**
+
+Describe any external requirements that might effect the issue (e.g need to await information from other subcomm).
+
+## **Related Issues/PRs:**
+
+Link any other issue or PR that might be related to this issue.
+
+## **Resources:**
+
+Link any resources or people that might be helpful in the accomplishment of the issue.
```

Comment:
Can you remove this one? As the person normally making the issues, I'd rather not be working off of a template